packages/venia-concept/_buildpack/create.js

```diff
@@ -0,0 +1,166 @@
+const { resolve } = require('path');
+
+function createProjectFromVenia({ fs, tasks, options }) {
+    const npmCli = options.npmClient;
+
+    const toCopyFromPackageJson = [
+        'main',
+        'browser',
+        'dependencies',
+        'devDependencies',
+        'optionalDependencies',
+        'engines'
+    ];
+    const scriptsToCopy = [
+        'buildpack',
+        'build:analyze',
+        'clean',
+        'download-schema',
+        'lint',
+        'prettier',
+        'prettier:check',
+        'prettier:fix',
+        'start',
+        'start:debug',
+        'test',
+        'validate-queries',
+        'watch'
+    ];
+
+    const scriptsToInsert = {
+        build:
+            'yarn run clean && yarn run validate-queries && webpack --no-progress --env.mode production',
+        'build:dev':
+            'yarn run clean && webpack --progress --env.mode development'
```

Comment:
Any reason why these are different than the ones in `venia-concept`? I feel like they should be identical.

Reply:
Because venia-concept needs to set itself up as part of the core dev experience. That's why it uses the `--core-dev-mode` flag of `buildpack load-env`. That's an internal, undocumented flag so that new clones of the repo can autogen a `.env` file where necessary.

This might be a little awkward! I'd be open to another approach to this--it just turned out to be tricky in the lifecycle to do this at repo root.

Reply:
@sirugh Spurred by your sensible concerns, I've removed the lockfiles from the PR for now, and opened https://github.com/magento/pwa-studio/issues/1631 to add them back later.